WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -49,10 +49,20 @@ extension NSNotification.Name {
     private static var zdClientId: String?
     private static var presentInController: UIViewController?
 
+    private static let sourcePlatform = "iOS"
     private static var appVersion: String {
         return Bundle.main.shortVersionString() ?? Constants.unknownValue
     }
 
+    private static var appLanguage: String {
+        guard let lang = Locale.current.languageCode else {
+            return ""
```

Comment:
Just one minor request - can we send `Constants.unknownValue` instead of an empty string? I think it helps indicate we _tried_, instead of just being blank and looking like a bug?